app/src/test/kotlin/com/intive/picover/auth/repository/AuthRepositoryTest.kt

```diff
@@ -151,5 +152,9 @@ class AuthRepositoryTest : ShouldSpec(
 			result shouldBeEqual Profile(userPhoto, "Marian Kowalski", userEmail)
 			slot.captured.displayName!! shouldBeEqual "Marian Kowalski"
 		}
+
+		should("not set userAvatarReference WHEN class is created") {
+			verify(inverse = true) { storageReference.child("user/$userUid") }
```

Comment:
I would rather simplify it and use regular `verify { mock wasNot Called }` and assert that any call to `storageReference` was performed, regardless of used function and passed arguments. 
If someone would change parameter of `child` call in `AuthRepository` and change it to ordinary property, then test would not detect it.

Reply:
@MarekMacko Updated

Reply:
Nice, thank you ;)